src/simple_openid_connect/integrations/django/views.py

```diff
@@ -50,11 +39,11 @@ class InitLoginView(View):
     """
 
     def get(self, request: HttpRequest) -> HttpResponse:
+        if "next" in request.GET.keys():
+            request.session["next"] = request.GET["next"]
```

Comment:
The django session is not scoped to this view (as the URL was). So maybe `login_redirect_uri` or something similar would be more appropriate than a generic `next` key in the session object,